Separate QGIS desktop and server documentation (#5930)
qgis/QGIS-Documentation: docs/user_manual/working_with_ogc/ogc_client_support.rst

.. _working_with_ogc:

> *********************
> Working with OGC data
> *********************

Reviewer: ```suggestion
***************************************
Working with ISO / OGC protocols
***************************************
```

Author: For now, I updated the documentation with *OGC standards* instead of protocols. Do you think ISO is necessary here?

Reviewer: > For now, I updated the documentation with _OGC standards_ instead of protocols. Do you think ISO is necessary here?

OGC developed the specifications (in cooperation with ISO TC211). ISO has published them as standards.

Author: Thanks for your detailed answer, I made the change with your suggestion

> **************************
> Working with OGC standards
> **************************

Reviewer: ```suggestion
***************************************
Working with OGC / ISO protocols
***************************************
```

specifications as a client, being **SFS** (through support of the PostgreSQL
/ PostGIS data provider, see section :ref:`label_postgis`).

> Note that you can also share your data through these OGC Services using a webserver

Reviewer: ```suggestion
You can also share your maps and data through the WMS, WMTS, WFS, WFS-T and WCS
using a webserver
```
But I guess that not all these protocols are supported by Mapserver, Geoserver or QGIS Server?

Author: Not sure, and the list could expand in the future. Better to keep *these OGC Services* in my opinion. 
I keep your suggestions for the start of the sentence though.

Reviewer: > Not sure, and the list could expand in the future. Better to keep _these OGC Services_ in my opinion.
> I keep your suggestions for the start of the sentence though.

1) WMS does not provide "data", it provides maps.
2) I still think it is better to mention the specific protocols. This is a live document.

Author: OK. Done

> with QGIS Server (see the :ref:`QGIS-Server-manual`), UMN MapServer or GeoServer installed.

Reviewer 2: Or ?
```suggestion
with :ref:`QGIS Server <QGIS-Server-manual>`, UMN MapServer or GeoServer installed.
```
